lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -15,6 +15,57 @@
  * @access private
  */
 class WP_Theme_JSON_6_1 extends WP_Theme_JSON_6_0 {
+	/**
+	 * Metadata for style properties.
+	 *
+	 * Each element is a direct mapping from the CSS property name to the
+	 * path to the value in theme.json & block attributes.
+	 */
+	const PROPERTIES_METADATA = array(
```

Comment:
This is added to the `6-1` file primarily to remove the `--wp--style--block-gap` CSS variable from being output. Styles at each level (as of this PR) should be output directly, without need of the CSS variable. For example, for Flex layouts, each level is now handled via classnames and specificity:

<img width="246" alt="image" src="https://user-images.githubusercontent.com/14988353/170641905-e7bda892-70ba-4100-8c48-f5dcaddd7917.png">